packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -1196,6 +1196,13 @@
       <description-ML>Minimum level to run partitioner. \muelu does not rebalance levels finer than this one.</description-ML>
     </parameter>
 
+    <parameter>
+      <name>repartition: use map</name>
+      <type>bool</type>
+      <default>false</default>
+      <description>Instead of using a matrix, use a map to determine rebalancing heuristics. Obviously, this cannot evaluate nonzero imbalance.</description>
```

Comment:
You could also use a weighted vector W = spones(A)*ones(size(A,1),1).

Reply:
True, that'd be a good addition. I won't be using it in RefMaxwell though, since we really don't have a matrix yet at this point, and I'm not sure if we can determine the nnz/row upfront.